cmd/nodecmd/create.go

```diff
@@ -0,0 +1,393 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package nodecmd
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"os/user"
+	"time"
+
+	"github.com/ava-labs/avalanche-cli/pkg/ansible"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/utils"
+	"github.com/hashicorp/hcl/v2/hclwrite"
+
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+
+	awsAPI "github.com/ava-labs/avalanche-cli/pkg/aws"
+	"github.com/ava-labs/avalanche-cli/pkg/terraform"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/spf13/cobra"
+)
+
+func newCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create [clusterName]",
```

Comment:
Can you add an "ALPHA" warning to both the SHORT and LONG fields here saying this is still undergoing testing?